### PR TITLE
Refactor actions with helper method and early returns

### PR DESCRIPTION
**Pull request by carmenquan**
To make our actions code cleaner, I refactored a few parts 

- Created a helper `ActionsHelper.getLastSelectedDefaultMutableTreeNode` which finds the item that was tapped. This was repeated across multiple actions
- Tried to early return as often as I could, easier to read imo
- Removed some constants and hard coded assumptions where possible (i.e. first child is always the flag key)
- Removed boilerplate comments
- Removed constructors, everything built fine without them and instead changing inheriting `AnAction` to constructor invocation

### CODE REVIEW

File: src/main/kotlin/com/launchdarkly/intellij/action/RunCoderefsAction.kt [hunk 1]
```diff
@@ -1,87 +0,0 @@
-package com.launchdarkly.intellij.action
```
Comment (carmenquan) [inline src/main/kotlin/com/launchdarkly/intellij/action/RunCoderefsAction.kt:1]: I deleted this file, its not used anywhere, we can always revive it in history if we need it.

Comment (jazanne) [reply src/main/kotlin/com/launchdarkly/intellij/action/RunCoderefsAction.kt:1]: 👀  does this mean we aren't _actually_ using the **Enable Code Refs** plugin settings? 😬

Comment (carmenquan) [reply src/main/kotlin/com/launchdarkly/intellij/action/RunCoderefsAction.kt:1]: I think we still are, this is for a code refs menu item to show up when you right click the flag list. The setting is hooked up to `FlagAliases` which is all hooked up.

Comment (jazanne) [reply src/main/kotlin/com/launchdarkly/intellij/action/RunCoderefsAction.kt:1]: AHH ok gotcha

**Review by jazanne — APPROVED** [review]

**Pull request merged**